### ISSUE TITLE
 NVDA Remote with "Attempt to cancel speech for expired focus events" speech fails

### ISSUE THREAD

**Issue by cary-rowen** [NVDARemote/NVDARemote#226]
This is a serious bug. I have already written an issue in the NVDA repo, but maybe I will report this bug to NVDA Remote, Then I will fully quote the contents of NVDA Repo.
Thanks.

---

### Steps to reproduce:
1. Both parties install NVDARemote V2.3 add-ons.
2. The controlled party opens the NVDA advanced settings panel and selects "Attempt to cancel speech for expired focus events" as Yes.
3. The initiating control party and the controlled party discuss the host and key information to be used for connection.
4. After hearing the beep of successful connection, the initiating control party presses F11 to try to operate the other party's computer.

### Actual behavior:
NVDA does not announce anything.

### Expected behavior:
NVDA should correctly read the content on the controlled party's computer, just like operating this machine.

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
NVDA2020.3RC1

#### Windows version:
Windows version: Windows 10 2004 (64-bit) build 19041.508

#### Name and version of other software in use when reproducing the issue:
NVDA Remote Version 2.3 

#### Other information about your system:
none

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NVDA2020.2 also has this Issue.

#### If addons are disabled, is your problem still occuring?
Yes.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes.

**Comment by alekssamos** [comment]
Hi. I have the same problem.

**Comment by cary-rowen** [comment]
Hi,
My friends, Is anyone paying attention to this issue?

**Comment by alekssamos** [comment]
> Go into NVDA's Advanced settings and set Attempt to cancel speech for expired focus events to Default on the remote machine. See if that fixes it.

**Comment by alekssamos** [comment]
#228

**Comment by cary-rowen** [comment]
It can indeed solve the problem, but this is only a temporary solution. This is not the best solution, because NVDA will set this option to "Yes" by default in the new version.
Hope someone can fix this error.
@alekssamos

**Comment by cary-rowen** [comment]
Hi,

Does the issue still exist? Will the new version resolve this issue?

Thanks

**Comment by fisher729** [comment]
Yes. This issue can be closed.

**Comment by cary-rowen** [comment]
The problem has been fixed, so close.
